sql/views.sql

```diff
@@ -259,5 +259,40 @@ CREATE OR REPLACE VIEW timescaledb_information.data_node AS
     LEFT OUTER JOIN LATERAL @extschema@.data_node_hypertable_info(CASE WHEN s.node_up THEN s.node_name ELSE NULL END) size ON TRUE
   GROUP BY s.node_name, s.node_up, s.owner, s.options;
 
+---compression parameters information ---
+CREATE VIEW timescaledb_information.compression_settings
+AS
+SELECT
+   ht.schema_name,
+   ht.table_name,
+   segq.attname,
+   segq.segmentby_column_index,
+   segq.orderby_column_index,
+   segq.orderby_asc,
+   segq.orderby_nullsfirst 
+FROM
+   (
```

Comment:
should get rid of subselect and also run pg_format on the query

Reply:
don't care for the sub-select myself. But is there any specific reason why it should not be used (didn't see any plan issues).